api/document/js/Api.ts

```diff
@@ -9,10 +10,10 @@ function setStatus(msg: string, className: 'editor-status-error'|'' = '') {
 }
 
 function setStatusError(e: Error) {
-  let errMsg = `An error occurred: ${e}`;
+  let errMsg = 'An error occurred.';
```

Comment:
I removed the exception stringification to reduce the verbosity of the error output; note that this _does_ mean that we won't be able to easily distinguish between a 400 and a 500 without looking at our developer tools' "network" tab, but I think that the actual content that follows will be more helpful than the status code in almost all cases.